ruleset/phpstan.neon

```diff
@@ -24,3 +25,4 @@ parameters:
         - '#Cannot call method [a-zA-Z0-9]+\(\) on Symfony\\Component\\Config\\Definition\\Builder\\NodeParentInterface\|null.#'
         - '#Call to an undefined method League\\Pipeline\\PipelineInterface\:\:process\(\).#'
         - '#Method Doctrine\\Persistence\\ObjectRepository<mixed>\:\:findOneBy\(\) invoked with 2 parameters, 1 required.#'
+        - '#Method Doctrine\\Persistence\\ObjectRepository<object>\:\:findOneBy\(\) invoked with 2 parameters, 1 required.#'
```

Comment:
maybe we should install `phpstan/doctrine-extension` ?

Reply:
Is the line just above still valid ?

Reply:
I modified the previous line that was no longer valid rather than adding this one.